meson.build

```diff
@@ -1,4 +1,4 @@
-project('gnome-shell', 'c',
+project('gnome-shell', ['c', 'cpp'],
```

Comment:
I'm actually not sure that the `cpp` bit is needed anymore. Could you test if it could be removed?

Reply:
Yes, it works.